Loop/AppDelegate.swift

```diff
@@ -129,6 +129,9 @@ final class AppDelegate: UIResponder, UIApplicationDelegate, DeviceOrientationCo
 
     func applicationWillTerminate(_ application: UIApplication) {
         log.default(#function)
+        
+        // if the app is being terminated, remove any workout override reminders
+        alertManager.retractAlert(identifier: LoopSettings.workoutOverrideReminderAlertIdentifier)
```

Comment:
I would remove this.  If the user has left this override turned on, why should it matter that the app is still running to get this reminder?